transifex/utils/common.py

```diff
@@ -0,0 +1,283 @@
+import logging
```

Comment:
There are no "non-common" utils. Simplify a bit and put this in `transifex/utils.py`.